File: code_blocks/integrations/third-party-integrations/clevertap_1.ts
import Purchases from '@revenuecat/purchases-react-native';
import CleverTap from 'clevertap-react-native';

async function syncCleverTapId() {
  await Purchases.configure({ apiKey: 'public_sdk_key', appUserID: 'my_app_user_id' });
  const cleverTapId = await CleverTap.getCleverTapID();
  await Purchases.setAttributes({ '$cleverTapId': cleverTapId });
}

File: code_blocks/integrations/third-party-integrations/clevertap_1.dart
import 'package:purchases_flutter/purchases_flutter.dart';
import 'package:clevertap_plugin/clevertap_plugin.dart';

Future<void> syncCleverTapId() async {
  await Purchases.configure(
    PurchasesConfiguration('public_sdk_key')..appUserId = 'my_app_user_id',
  );
  final cleverTapId = await CleverTapPlugin.profileGetCleverTapID();
  await Purchases.setAttributes({'\$cleverTapId': cleverTapId});
}

